assets/script.js

```diff
@@ -116,6 +116,10 @@ function getAlbumInfo() {
         //album search result shown
         $("#album-pic").attr("src", icon);
         $("#summaryHeading").text(albumName);
+        $("#album-pic").attr("title", response.album.artist);
+        $("a.img").each(function () {
+          $(this).attr("title", $(this).find("img").attr("title"));
+        });
```

Comment:
Comment exactly what this is doing as it's not the most readable line

Reply:
bug fixed